### ISSUE TITLE
Support for Meteor 3

### ISSUE THREAD

**Issue by diavrank** [copleykj/socialize-user-presence#26]
Hi, I am upgrading my app to Meteor 3 and realized that this package is not compatible. Are there plans to add support of this package to version 3?

**Comment by StorytellerCZ** [comment]
Yes, I will have PR in due time. For most of the other socialize packages I have already posted them.

**Comment by StorytellerCZ** [comment]
Updated my PR #24 with Meteor 3 support
